astropy/convolution/convolve.py

```diff
@@ -641,3 +643,33 @@ def convolve_fft(array, kernel, boundary='fill', fill_value=0, crop=True,
         return result
     else:
         return rifft.real
+
+def convolve_models(model_a, model_b, mode='convolve_fft'):
+    """
+    Convolve two models using `~astropy.convolution.convolve_fft`.
+
+    Parameters
+    ----------
+    model_a : `~astropy.modeling.core.Model`
+        First model
+    model_b : `~astropy.modeling.core.Model`
```

Comment:
While mathematically `model_a` and `model_b` are interchangeable in the convolution, it still might make sense to declare one of them as the `kernel`, because in practice (e.g. when used to model instrument response functions ) it has a few special properties:

- Normalization (typically integrates to unity)
- Centered on the array of coordinates it's evaluated on
- Fixed parameters when fitted